### DISCUSSION TITLE
unable to login Github' website if a github app in phone

### DISCUSSION THREAD

**Discussion by LeoChen98** [community/community#143029] [Mobile]
as the title, i failed to login the github website when i have a github app on phone. everytime i pressed 'login', the website trys to call the a app up but not included github app itself. and whatever i choose in that popup, the website never redirect to next step though i approved the loging in app.

![Screenshot_2024-10-30-09-43-55-397_android.jpg](https://github.com/user-attachments/assets/19f392c7-f32c-45d8-b3f8-039f2c20149d)

**Comment by eliperkins** [comment]
Hey @LeoChen98, thanks for giving us the heads up on this! Can I get some more information from you to address this?

- You mentioned `i failed to login the github website when i have a github app on phone`. Does uninstalling the GitHub Mobile app make this bug go away?
- What Android device and OS are you running?
- What browser is your default browser?

**Reply by LeoChen98** [reply]
![Screenshot_2024-10-31-01-33-38-946_com.android.settings.jpg](https://github.com/user-attachments/assets/7fbaf638-1344-4f40-8c6c-0051df098e56)

the os is based on Android 14

and browser is chrome @ 130.0

**Reply by LeoChen98** [reply]
cannot try the first. i need the app for two-factor login. once i uninstall it, i cannot login anymore.

however, the  bug won't occur on my another phone without the app.

**Comment by Terminal127** [comment]
first go in your github app and then try to signin form there , is possible use desktop mode in the browser and see if it is fixed or not .

**Reply by LeoChen98** [reply]
it is desktop mode already. neither desktop mode or mobile mode can sign in

**Comment by areteruhiro** [comment]
Hello, I was also having this problem with Hyper OS. Here's how to solve it: 
1.Open the device's settings app 
2.Touch the "Apps" item
 3.Touch Manage apps 
  4.Touch the ︙ in the upper right 
  5.Other settings
  6.Default apps 
  7.Open links 
  8.Touch Git hub 
  9.Don't open compatible links Don't forget to change it back to "Open" after logging in. (This method is temporary, and ideally the app will be compatible with HyperOS.)
  
This sentence uses Google Translate.


ddepl↓
Hi, I also had this problem with Hyper OS. The solution is as follows. (1) Open the Settings app on the device (2) Touch the "Apps" item (3) Touch Manage Apps (4) Touch ︙ in the upper right corner (5) Other settings (6) Default app (7) Open link (8) Touch Git hub (9) Do not open the corresponding link. (Do not forget to return to "Open" after logging in. (This method is tentative and it is ideal if your app supports HyperOS.)

Translated with DeepL.com (free version)

**Reply by areteruhiro** [reply]
Also, this issue only occurs in Chrome, not in Fire Fox.

**Reply by hfloyd** [reply]
Thanks for this, it fixed my issue when using GitHub as the login provider for a different website. ❤️

**Comment by rolddevs** [comment]
Two-Factor Authentication (2FA) or maybe it is Browser Cache

**Reply by trane77** [reply]
Nope, I just tried to clear the cache, login with a different browser: same result.

You know, maybe it's just random, but after opening the GitHub mobile app and enabling biometric access (with keep access for: 30 minutes, to be precise 😄), I logged in from web successfully.
It showed the "select where to open" modal, but this time the login page proceeded to the next step, where I selected GitHub mobile to complete the 2FA.

Honestly I don't know why this happens, but I'm using HyperOS as well, so I think there's some kind of problem with the app URL scheme of the  GitHub mobile app.

**Comment by ghost** [comment]
Ok

**Comment by guymarshall** [comment]
> as the title, i failed to login the github website when i have a github app on phone. everytime i pressed 'login', the website trys to call the a app up but not included github app itself. and whatever i choose in that popup, the website never redirect to next step though i approved the loging in app.
> 
> ![Screenshot_2024-10-30-09-43-55-397_android.jpg](https://github.com/user-attachments/assets/19f392c7-f32c-45d8-b3f8-039f2c20149d)
> 
> 

I also cannot login using Firefox. In mobile or desktop view it just opens the popup and whatever I choose (or if I dismiss it) nothing happens. I just wish GitHub would keep me logged in for more than 5 days -_-

**Comment by tagir404** [comment]
try opening this link during the login process
 https://github.com/sessions/verified-device/mobile?auto=true

the code should be displayed there

**Comment by Armmer-compa** [comment]
Could you please specify what kind of answer you're looking for? Are you referring to a specific question, topic, or situation? Let me know so I can provide the best possible response.

**Comment by 1Thef0x** [comment]
you can hold the link and open in new tab easy way 
or use another browser like arc

**Comment by m00nbyt3** [comment]
Same problem here. Im using a Realme with Realme UI 6.0

**Reply by m00nbyt3** [reply]
Oh, just updated the app to version 1.225.0 (last one to this day) and problem solved
